app/src/main/java/de/danoeh/antennapod/fragment/AudioPlayerFragment.java

```diff
@@ -515,6 +516,8 @@ public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
             } else {
                 txtvSeek.setText(Converter.getDurationStringLong(position));
             }
+        } else if (controller.getDuration() != controller.getMedia().getDuration()) {
+            updateUi(controller.getMedia());
```

Comment:
Doesn't this mean that it will update the UI every single second when the progress is updated?

Reply:
It does. Forgot something there :D. Now, it should only update if the duration changed

Reply:
Hmm. This feels a bit risky, to be honest. The same media object instance is passed to all fragments. If the UI starts to write to the object, we might get unexpected side effects in other fragments.

Reply:
Okay I see. What to you think about this solution? Maybe we could use this variable in other places too.